admin/workspace-providers/deployment.md

```diff
@@ -106,7 +108,8 @@ when communicating with the Coder deployment.
       --install \
       --force \
       --set envproxy.token=[REMOTE_ENVPROXY_TOKEN] \
-      --set ingress.host=[HOSTNAME] \
+      --set envproxy.accessURL=[HOSTNAME] \
+      --set ingress.host=[HOSTNAME_WITH_NO_PROTOCOL] \
       --set envproxy.cluster-address=[CLUSTER_ADDRESS] \
```

Comment:
observation: we sometimes use hyphens and sometimes use camelCase (e.g. accessURL)